config/sidebars.js

```diff
@@ -238,7 +238,9 @@ const cli = [
   ...sidepageHeader("CLI"),
   ,
   "cli/README",
+  ...section("Commands"),
   "cli/commands",
```

Comment:
Did you want to add the `shell.md` here, or not yet? Otherwise, it won't render in the sidebar.

Reply:
Fwiw, I did want to make each command its own page, so your format def aligns to that approach once decomposed—in case that was the intent.

Reply:
Yep! I actually made in issue for that exact thing a few days ago, DOC-34.

Reply:
> @awmuncy left a couple other comments but otherwise lgtm

Updated the sidebar thing, we can defer moving all commands into their own pages